securing_apps/topics/oidc/nodejs-adapter.adoc

```diff
@@ -257,6 +257,9 @@ By default, the middleware catches calls to `/logout` to send the user through a
 configuration parameter to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { logout: '/logoff' } ));
+    
+When the user-triggered logout is invoked a query parameter `redirect_url` can be passed. 
+This parameter is then used as the redirect url of the OIDC logout endpoint.
```

Comment:
@bender316 thanks for submitting this. My suggestion is to not only include this paragraph, but also a snippet with examples. What do you think?